### PR TITLE
implement advantages section desktop& tablet #8

### LINKED ISSUE #8: create-and-style-for-advantages-section

### PR DESCRIPTION
**Pull request by 2H1EU8**
image
image
image

### CODE REVIEW

File: src/stylesheets/pages/home/_results.scss [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+    p:first-child {
+      margin-bottom: 20px;
+    }
+
+
+  }
+
+  &__circle {
+    position: relative;
+    width: pxToRem(204);
```
Comment (milo219) [inline src/stylesheets/pages/home/_results.scss:84]: check width and height

File: src/stylesheets/pages/home/_results.scss [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        border-width: 5px ;
+      }
+    }
+
+    .progress-right {
+      right: 1;
+
+      .progress-bar {
+        left: -100%;
+        border-top-left-radius: 204px;
```
Comment (Lanng21) [inline src/stylesheets/pages/home/_results.scss:137]: Limit using sizes over 100px

File: src/stylesheets/pages/home/_results.scss [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+        -webkit-transform-origin: center right;
+      }
+    }
+
+    .progress-left {
+      left: 0;
+
+      .progress-bar {
+        left: 100%;
+        border-top-right-radius: 204px;
```
Comment (Lanng21) [inline src/stylesheets/pages/home/_results.scss:150]: the same

File: src/layout/main.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+          </div>
+          <div class="results__text">
+            <p class="title--secondary upper-case">UNIQUE VISITORS</p>
+            <p class="decs">WordPress, the premier free open-source blogging utility, has gone through several upgrades in its life.</p>
+          </div>
+        </div>
+      </div>
+
+    </div>
+  </section>
```
Comment (Lanng21) [inline src/layout/main.html]: move section to component. main not clear

File: src/components/results-sec.html [hunk 5]
```diff
@@ -0,0 +1,59 @@
+<section class="results">
+  <div class="container results__container">
+
+    <div class="results__title">
+      <h1 class="title--primary">Stunning Results</h1>
+      <p class="decs">Just look to the statistics we have generate the past year.</p>
+    </div>
+
+    <div class="results__content">
+      <div class="results__col">
```
Comment (Lanng21) [inline src/components/results-sec.html]: split components. i see it repeating a lot

File: src/stylesheets/pages/home/_results.scss [hunk 6]
```diff
@@ -0,0 +1,205 @@
...
+      line-height: pxToRem(108);
+      margin-bottom: 15px;
+    }
+
+    &::after {
+      content: "";
+      width: 100%;
+      height: 100%;
+      border-radius: 50%;
+      border: 1px solid #DADADA;
```
Comment (Lanng21) [inline src/stylesheets/pages/home/_results.scss]: color here?

**Pull request merged**